storage/jellyfish_merkle/src/node_type/node_type_test.rs

```diff
@@ -81,6 +82,29 @@ fn test_encode_decode() {
     }
 }
 
+#[test]
+fn test_internal_validity() {
+    assert!(panic::catch_unwind(|| {
```

Comment:
You can use the `should_panic` annotation on the test.

Reply:
But I have two tests inside that will panic.